cloud/git_test.go

```diff
@@ -66,14 +66,14 @@ func TestNormalizeGitURL(t *testing.T) {
 			normalized: "something else",
 		},
 		{
-			name:       "unrecognized ssh url",
+			name:       "unrecognized ssh url - THIS IS A BUG IN THE go-gh library",
```

Comment:
@mariux this is a bug that should never happen in the real world (I guess). Only if someone place an incorrect git URL in the remote configuration. 
The problem is that `git://` scheme does not support port numbers in the URL, in then the go-gh library parses it incorrectly. In order to use port number, the user must provide the `ssh://` protocol (or the `ssh+git://` protocol).

Reply:
hmmm can the go-gh library parse gitlab and co?

Reply:
yes, just added tests covering that! Thanks